Jagrati/settings/base.py

```diff
@@ -118,3 +119,12 @@
 
 
 AUTH_USER_MODEL = 'accounts.User'
+
+AUTHENTICATION_BACKENDS=['django.contrib.auth.backends.ModelBackend',
+                         'social_core.backends.google.GoogleOAuth2']
+
+ALLOWED_HOSTS=['jagrati.herokuapp.com','localhost','127.0.0.1']
```

Comment:
We don't need this line as for development, allowed hosts are already specified in `development.py` and for production, we fetch allowed hosts from environment variables.